.github/workflows/presubmit.yml

```diff
@@ -69,7 +69,9 @@ jobs:
         run: echo $DIST_DIR && ls $HOME/Library/Android/sdk
       - name: build on server
         run: |
-          cd room && ./gradlew :buildOnServer && cd ../work && ./gradlew :buildOnServer
+          cd paging && ./gradlew :buildOnServer
```

Comment:
this wouldn't work since you won't end up in the right folder.

Reply:
Fixed now.